docs/release-notes/8.4.asciidoc

```diff
@@ -1,6 +1,20 @@
 [[release-notes-header-8.4.0]]
 == 8.4
 
+[discrete]
+[[release-notes-8.4.2]]
+=== 8.4.2
+
+[discrete]
+[[bug-fixes-8.4.2]]
+==== Bug fixes and enhancements
+* Removes access to the **Notes** and **Pinned** tabs in Timeline templates ({pull}140478[#140478]).
+* Fixes a bug with the **Attach to existing case** option in Timeline ({pull}139929[#139929]).
+* Fixes bugs in the Rules table that affected the selected rule count and bulk select feature ({pull}139461[#139461]).
+* Fixes a bug that caused the Rules page to incorrectly notify users that a prebuilt rules update was available, even after rules were fully updated ({pull}139287[#139287]).
+* Fixes a bug that prevented users from accessing alert details if they didn't have the appropriate privileges to view the internal index `.internal.alerts-security.alerts-spaceId`. Now, the Alert details flyout correctly uses the public alias index `.alerts-security,akerts-spaceId` ({pull}138331[#138331]).
```

Comment:
Couple of typos -- I guessed on the comma/period based on the linked PR might want to confirm with dev:
```suggestion
* Fixes a bug that prevented users from accessing alert details if they didn't have the appropriate privileges to view the internal index `.internal.alerts-security.alerts-spaceId`. Now, the Alert details flyout correctly uses the public alias index `.alerts-security.alerts-spaceId` ({pull}138331[#138331]).
```

Reply:
Pretty sure your corrections are right. @michaelolo24 would you mind confirming?

Reply:
Yep, it looks good. Thanks!

Reply:
We typically use `<Kibana-space>` , for example: https://www.elastic.co/guide/en/security/master/security-spaces.html